### PR TITLE
Fix displaying product thumbnails

### PR DESCRIPTION
**Pull request by molka**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | When i clicked on the "quick view" link and the modal is opened, the product thumbnails was not displayed correctly.
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-2739
| How to test?  | On small devices (nearly between 1040px and 768px ), Click on "quick view" link, you can see that the product thumbnails are displayed correctly

### CODE REVIEW

**Comment by vincentbz** [comment]
Hi @molka 

Could you move your PR to the 1.7.1.x branch please ? so that it will be released in the next patch version (1.7.1.1) and not 1.7.2.0.

Thanks !

**Comment by kpodemski** [comment]
@vincentbz this can be done by maintainer, @aleeks or someone else

**Comment by aleeks** [comment]
All svg are needed ?! :open_mouth:

**Comment by maximebiloe** [comment]
@aleeks: Yes they are

**Comment by Quetzacoalt91** [comment]
The pull-request was outdated, I've just rebased it on `develop` to remove conflicts.

**Review by Quetzacoalt91 — DISMISSED** [review]

**Review by eternoendless — DISMISSED** [review]
Code looks good!

File: themes/classic/assets/css/theme.css [hunk 1]
```diff
@@ -4,4 +4,10 @@
...
...-color:#4cbb6c;border-color:#4cbb6c}.card-success .card-footer,.card-success .card-header{background-color:transparent}.card-info{background-color:#5bc0de;border-color:#5bc0de}.card-info .card-footer,.card-info .card-header{background-color:transparent}.card-warning{background-color:#ff9a52;border-color:#ff9a52}.card-warning .card-footer,.card-warning .card-header{background-color:transparent}.card-danger{background-color:#ff4c4c;border-color:#ff4c4c}.card-danger .card-footer,.card-danger .card-header,.card-outline-primary{background-color:transparent}.card-outline-primary{border-color:#2fb5d2}.card-outline-secondary{background-color:transparent;border-color:transparent}.card-outline-info{background-color:transparent;border-color:#5bc0de}.card-outline-success{background-color:transparent;border-color:#4cbb6c}.card-outline-warning{background-color:transparent;border-color:#ff9a52}.card-outline-danger{background-color:transparent;border-color:#ff4c4c}.card-inverse .card-footer,.card-inverse .card-header{border-color:hsla(0,0%,100%,.2)}.card-inverse .card-blockquote,.card-inverse .card-footer,.card-inverse .card-header,.card-inverse .card-title{color:#fff}.card-inverse .card-blockquot
```
Comment (eternoendless) [inline themes/classic/assets/css/theme.css]: This merge conflict *must* be addressed

**Review by eternoendless — CHANGES_REQUESTED** [review]

File: themes/classic/assets/css/theme.css (hunk 1 above)
Comment (maximebiloe) [reply themes/classic/assets/css/theme.css]: It should be okay now 😉

**Review by eternoendless — APPROVED** [review]
Thanks @maximebiloe :)

**Comment by aleeks** [comment]
Thank you @molka & @maximebiloe

**Pull request merged**